pipelines/data_embedding_pipeline/config_data_embedding_pipeline.yaml

```diff
@@ -0,0 +1,11 @@
+steps:
+  fetch_data:
+    enable_cache: False
+    parameters:
+      data_version: "v1.0"
```

Comment:
I wonder if we'd be able define this once rather than twice to prevent inconsistencies?

Reply:
Can't find any way to put it as a parameter to the pipeline. We could return it from `fetch_data`:
```
    mind_df, nhs_df, data_version = fetch_data()

    mind_df = embed_data(mind_df, data_version)
    nhs_df = embed_data(nhs_df, data_version)
```

Reply:
Yeh a potential option, happy with what's already in there.

It does appear that `steps` has a `parameter` option, see here: https://docs.zenml.io/user-guide/advanced-guide/configure-steps-pipelines#an-example-of-a-yaml-config